extending-valtimo/extending-valtimo.md

```diff
@@ -1,5 +1,9 @@
 # Extending Valtimo
 
+This section contains information on how to use Valtimo features for which additional code is required. These are 
```

Comment:
Maybe rephrase this like:

This section contains instructions and examples on how to extend functionality found in Valtimo, generally through the use of code.